crates/authentication/README.md

```diff
@@ -30,43 +30,63 @@ To use this crate on Android, you must add the following to your app's `AndroidM
 <uses-permission android:name="android.permission.USE_BIOMETRIC" />
 ```
 
-## Example
-
-```rust
-use robius_authentication::{
-    AndroidText, BiometricStrength, Context, Policy, PolicyBuilder, Text, WindowsText,
-};
-
-let policy: Policy = PolicyBuilder::new()
-    .biometrics(Some(BiometricStrength::Strong))
-    .password(true)
-    .companion(true)
-    .build()
-    .unwrap();
-
-let text = Text {
-    android: AndroidText {
-        title: "Title",
-        subtitle: None,
-        description: None,
-    },
-    apple: "authenticate",
-    windows: WindowsText::new("Title", "Description"),
-};
-
-let callback = |auth_result| {
-    match auth_result {
-        Ok(_)  => log::info!("Authentication success!"),
-        Err(_) => log::error!(Authentication failed!"),
-    }
-};
-
-Context::new(())
-    .authenticate(text, &policy, callback)
-    .expect("Authentication failed");
+## Usage on Linux
+
+On Linux, `robius-authentication` uses **polkit** to request authorization via the
+desktop environment's native authentication prompt (GNOME/KDE/etc).
+
+> [!IMPORTANT]
+> **Ensure a polkit agent is running**
+>
+> The prompt is displayed by a polkit authentication agent (GNOME/KDE usually start one automatically).
+> If no agent is running (headless/SSH), no prompt will appear and auth will fail.
+
+### Write policy file.
+
+You can crate your own application's policy file, also can crate by template policy file.
+
+Template policy file see: [`./examples/org.robius.authentication.policy`](./examples/org.robius.authentication.policy)
+
+### Quick Test Mode ⚠️
+
+Add the policy file to actions by manually executing the following command:
+
+```bash
+sudo install -Dm644 com.yourapp.policy /usr/share/polkit-1/actions/
+```
+
+Then, ensure your policy file is correctly installed.
+
+```bash
+pkaction --action-id <YOUR_POLICY_File_ACTION_ID>
 ```
 
-For more details about the prompt text, see the `Text` struct,
-which allows you to customize the prompt for each platform.
+> During the test mode, you don't need to worry about the location of the policy file; just ensure it installs correctly.
+>
+> You can also store it in advance under the unified packaging configuration folder (like `packaging`)to facilitate automatic installation of the policy file during release mode when users perform installation.
```

Comment:
what does "store it in advance" mean? As in, create it in advance and move/install it into the proper directory _before_ installing the application? 

I feel like this sentence doesn't really help anyone; what were you hoping to convey here? Is this for developers or for users? or perhaps for people who are packaging up an app?

Missing a space after "(like `packaging`)"